apps/src/templates/currentUserRedux.js

```diff
@@ -213,6 +215,9 @@ export default function currentUser(state = initialState, action) {
       progressTableV2ClosedBeta: progress_table_v2_closed_beta,
       isLti: is_lti,
       isTeacher: user_type === UserTypes.TEACHER,
+      dateProgressTableInvtationDelayed:
```

Comment:
nit: Invtation -> Invitation
```suggestion
      dateProgressTableInvitationDelayed:
```